lib/plugins/rest.js

```diff
@@ -27,6 +27,11 @@ exports.register = function (server, options, next) {
 		throw error;
 	}
 
+	if (process.env.AWS_DEFAULT_SECURITY_GROUP === undefined) {
```

Comment:
I don't think this needs to be a required parameter. We can provide a hard-coded default, and optionally allow the user to customize the default via the variable.